### DISCUSSION TITLE
Invoke MPF on startup with Ubuntu

### DISCUSSION THREAD

**Discussion by ohmbrew** [missionpinball/missionpinball#75] [Q&A]
Hello! Does anyone use a reliable method to startup MPF when Ubuntu boots? I have tried various methods I found on the internet:

- cron job
- using Ubuntu's Startup Application interface

But none of them seem to work. Anyone care to share their method?

Greg

**Comment by Pucksnok** [comment]
Hi.
I did this a few weeks ago with gnome-startup-applications, using a bash script that launch mpf. Works great with Ubuntu 20.04. 


My script looks like this...
``
#!/bin/bash 
 # Script for running production build
 mpf both -P
``

/Gill

**Reply by ohmbrew** [reply]
Thanks Gill. Your script is all on one line but I think I see what you're doing. How does the script know to invoke mpf from a machine's root folder?

Greg

**Reply by Pucksnok** [reply]
Sorry, were a bit fast with copy paste. Here's the startup script I use... :D

#!/bin/bash 
 xterm -e "cd /home/pinball/Pokemon-Pinball/ && mpf both -p"

**Reply by ohmbrew** [reply]
ah. I hadn't used the xterm command in prior attempts using a bash script, just invoking the mpf executable. Thanks, I'll try this.

**Reply by ohmbrew** [reply]
Thanks Gill! I have it working now.
